uPortal-webapp/src/main/java/org/apereo/portal/context/rendering/RenderingPipelineConfiguration.java

```diff
@@ -146,6 +146,9 @@
     @Value("${org.apereo.portal.layout.useFlyoutMenus:false}")
     private String useFlyoutMenus;
 
+    @Value("${org.apereo.portal.layout.faviconPath}")
```

Comment:
I suggest a default here, like... `@Value("${org.apereo.portal.layout.faviconPath:/favicon.ico}")`.

(More on this suggestion below.)

Reply:
I didn't do that to avoid to apply the Spel, as the property (if we want to keep the actual path) depends as we should add the app context like : ${portal.context}/favicon.ico

Reply:
After change the default value is null and so the parameter is passed to the xsl only if set from `uPortal.properties` (or `global.properties`)